desk/app/expose.hoon

```diff
@@ -0,0 +1,2233 @@
+        justify-content: space-between;
+        align-items: center;
+      }
+
+      @media screen and (min-width: 960px) {
+        body.chat .prelude {
+          margin: 0;
+          position: fixed;
+          bottom: 2em;
+          left: 2em;
+          border: 0;
+          background: var(--background);
+          font-size: 0.75em;
+          padding: 0.5em 0.75em;
+          border-radius: 0.5em;
+          gap: 1em;
+        }
+      }
+
+      .prelude .author {
+        display: flex;
+        align-items: center;
+        gap: 0.5em;
+        font-weight: 500;
+        color: var(--text-muted);
+      }
+
+      .prelude .author .avatar {
+        border-radius: 0.25em;
+        overflow: hidden;
+      }
+
+      .prelude .author .avatar img {
+        width: 1.5625em;
+        height: 1.5625em;
+        object-fit: cover;
+      }
+
+      .prelude time {
+        color: var(--text-muted);
+        padding: 0;
+      }
+
+      article p:empty {
+        display: none;
+      }
+
+      article img {
+        max-width: 100%;
+        border-radius: 0.5em;
+      }
+      '''
+    ::
+    ++  heads
+      |=  [title=tape img=(unit @t)]
+      ;head
+        ;title:"{title}"
+        ;style:"{(trip style)}"
+        ;link(rel "preconnect", href "https://rsms.me");
+        ;link(rel "stylesheet", href "https://rsms.me/inter/inter.css");
+      ::
+        ;meta(charset "utf-8");
+        ;meta(name "viewport", content "width=device-width, initial-scale=1");
+      ::
+        ;meta(name "robots", content "noindex, nofollow, noimageindex");
+      ::
+        ::REVIEW  make sure this is the right/new app id
+        ;meta(property "apple-itunes-app", content "app-id=6451392109");
+        ::NOTE  at the time of writing, android supports no such thing
+      ::
+        ::TODO  could get even smarter about description, preview image, etc
+        ;meta(property "og:title", content title);
+        ;meta(property "twitter:title", content title);
+        ;meta(property "og:site_name", content "Tlon");
+        ;meta(property "og:type", content "article");
+        ;meta(property "og:article:author:username", content (scow %p author.msg));
+      ::
+        ;*  ?~  img
+            :_  ~
+            ;meta(property "twitter:card", content "summary");
+        =/  img=tape  (trip u.img)
+        :~  ;meta(property "twitter:card", content "summary_large_image");
+            ;meta(property "og:image", content img);
+            ;meta(property "twitter:image", content img);
+        ==
+      ::
+        ;*  ?~  aco  ~
+            ?:  =('' nickname.u.aco)  ~
+            :_  ~
+        ;meta(property "og:article:author:first_name", content (trip nickname.u.aco));
+      ==
+    ::
+    ++  footer
+      ;footer
+        ;a(href "https://tlon.io")
+          ;img@"https://tlon.io/icon.svg"(alt "Tlon logo", width "18");
+          ;span
+            ; Powered by Tlon
+          ==
+        ==
+      ==
+    ::
+    ++  chat-prelude
```

Comment:
these preludes seem like they could just be smashed together into one gate if you make the arg unit